### PR TITLE
SetBuiltinFunctions and SetBuiltinModules

### PR DESCRIPTION
**Pull request by earncef**
@d5 I still need to review all the changes but the feature is working.  Please review as well.

#119

### CODE REVIEW

File: runtime/vm.go [hunk 1]
```diff
@@ -1414,11 +1425,11 @@ func indexAssign(dst, src *objects.Object, selectors []*objects.Object) error {
 }
 
 func init() {
```
Comment (d5) [inline runtime/vm.go]: We should remove this function.

File: script/script.go [hunk 2]
```diff
@@ -56,22 +56,30 @@ func (s *Script) Remove(name string) bool {
 	return true
 }
 
-// DisableBuiltinFunction disables a builtin function.
-func (s *Script) DisableBuiltinFunction(name string) {
-	if s.removedBuiltins == nil {
-		s.removedBuiltins = make(map[string]bool)
+// SetBuiltinFunctions allows to define builtin functions.
+func (s *Script) SetBuiltinFunctions(funcs map[string]*objects.BuiltinFunction) {
```
Comment (d5) [inline script/script.go]: I think this should probably be `SetBuiltinFunctions([]*objects.BuiltinFunction)`. I forgot that BuiltinFunction had Name when suggested this signature in the issue 😄.

**Comment by d5** [comment]
Also, can you update [documentation](https://github.com/d5/tengo/blob/master/docs/interoperability.md#sandbox-environments) accordingly?

**Comment by earncef** [comment]
Sure

**Comment by d5** [comment]
This will resolve #119

**Comment by earncef** [comment]
All done.

**Pull request merged**